test-utils-core/src/test/java/uk/gov/justice/services/test/utils/core/rest/RestClientTest.java

```diff
@@ -94,6 +97,19 @@ public void shouldSendQueryWithHeaders() {
         assertThat(response.getStatus(), org.hamcrest.CoreMatchers.equalTo(OK_STATUS_CODE));
     }
 
+    @Test
+    public void shouldSendDelete() {
+
+        stubFor(delete(urlEqualTo(RESOURCE_PATH))
+                .withHeader(CONTENT_TYPE, equalTo(CONTENT_TYPE_VALUE))
+                .withHeader(HEADER, equalTo(HEADER_VALUE))
+                .willReturn(aResponse().withStatus(NO_CONTENT_CODE)));
+
+        Response response = restClient.deleteCommand(URL, CONTENT_TYPE_VALUE, getHeaders());
+
+        assertThat(response.getStatus(), org.hamcrest.CoreMatchers.equalTo(NO_CONTENT_CODE));
```

Comment:
CoreMatchers can be static imported

Reply:
The static import of CoreMatchers.equalTo would conflict with the existing static import of Wiremock.equalTo